Documentation/cmdref/cilium-agent.md

```diff
@@ -18,6 +18,7 @@ cilium-agent [flags]
       --allow-localhost string                               Policy when to allow local stack to reach local endpoints { auto | always | policy } (default "auto")
       --annotate-k8s-node                                    Annotate Kubernetes node (default true)
       --api-rate-limit map                                   API rate limiting configuration (example: --rate-limit endpoint-create=rate-limit:10/m,rate-burst:2) (default map[])
+      --arping-refresh-period duration                       Period for ARP entries to remote nodes refresher (default 5m0s)
```

Comment:
Does it make sense to make it possible to disable refresh completely if hw addrs don't change in most clusters? E.g. give a zero or negative duration?

Reply:
That's a good idea, will do that!